code/implementation/docker/machine/src/main/resources/schema/base/digitaloceanConfig.json.d/digitalocean.json

```diff
@@ -0,0 +1,20 @@
+{
+    "resourceFields": {
```

Comment:
No, they don't. Not 100% sure why. Possibly because their parent is tagged with list = false?
https://github.com/rancherio/cattle/pull/143/files#diff-408ea3c153e76fa0049e58a7953d5ba6R6